### ISSUE TITLE
add rel="nofollow" to richtext editor links

### ISSUE THREAD

**Issue by yoann54** [storyblok/storyblok#360]
**The feature would affect:** (check one with "x")   
- [x ] *app.storyblok.com (CMS - Interface)*   
- [x ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**
I can't add a rel="nofollow" on the link i add in the rich text editor.

**Describe the solution you'd like**
In the rich-text editor it would be usefull to add a checkbox to add a rel="nofollow" on the link.

**Describe alternatives you've considered**
if i could add a class on some of the links i could have treated this problem in my front with some thing like that :
mounted () {
    let links = document.querySelectorAll('a[href]:not([href^=mailto]):not([class^=addfollow])')
      console.log(links)
    for (let i = 0; i < links.length; i++) {
      if (links[i].hostname != window.location.hostname) {
        links[i].target = '_blank'
        links[i].rel = 'noopener nofollow noreferrer'
      }
    }
  }

**Additional context**

**Comment by einomi** [comment]
This feature would be helpful for us too.

**Comment by joaomarcelofm** [comment]
This is also a feature our team is looking for.

Beyond `rel` there are other attributes that would be beneficial to have, such as `Title` and `Class` so we can pass it on to the link to enrich the SEO and add event tiggers based on the classes passed to the link.

@ShivamChaurasia @nachoadjust

**Comment by bstrdChronicles** [comment]
I totally agree that is a feature that i had to have separate fields in my link component for. 
In Essence though, things like these should be included in the "link" field type that is offered by Storyblok. :)

Or maybe an option could be added that allows the addition of boolean fields or selections in the rich text editor field configuration ?

Also it would really help in working with the field type in components and the rich text editor if the component field type "link" and the link inside a rich text field would have the same options and can be configured in the same way. 
e.g. linked component restriction etc. 

Thanks a lot!

**Comment by julia-utenkova** [comment]
Voting for this feature to be implemented!

**Comment by Phil404** [comment]
We also need this feature :)

We tried to use a custom component for this feature, but this worked not for us.

Thank you very much :)

**Comment by mxmtsk** [comment]
The ability to add `rel` to links is a bare minimum for any larger SEO driven site, such as magazines, news websites and blogs.

Here's some more information by Google regarding this attribute:

https://developers.google.com/search/blog/2019/09/evolving-nofollow-new-ways-to-identify
https://developers.google.com/search/docs/advanced/appearance/qualify-outbound-links?hl=de&visit_id=637686263359524674-2190281451&rd=1

**Comment by yanchenhao57** [comment]
We really need this feature, hope you can add it soon🙏

**Comment by alvarosabu** [comment]
Hi @yoann54, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
